requirements-base.txt

```diff
@@ -84,6 +84,7 @@ brotli>=1.0.9
 pyuwsgi==2.0.23
 zstandard>=0.18.0
 sentry-usage-accountant==0.0.10
+orjson==3.10.0
```

Comment:
these should all use `>=` -- we don't install from this file we use it to generate the pins so it should be as permissive as possible